sql/scan.go

```diff
@@ -725,8 +731,8 @@ func (n *scanNode) prettyKey() string {
 
 func prettyKeyVals(vals []parser.Datum) string {
 	var buf bytes.Buffer
-	for _, v := range vals {
-		fmt.Fprintf(&buf, "/%v", v)
+	for _, val := range vals {
+		fmt.Fprintf(&buf, "/%s", val)
```

Comment:
The renaming is gratuitous, but harmless. I sometimes prefer `%v` to `%s` because it prints `nil` nicer, though I don't think that matters here.